documentation/book/master.adoc

```diff
@@ -18,7 +18,8 @@ include::assembly-using-the-user-operator.adoc[leveloffset=+1]
 
 include::assembly-security.adoc[leveloffset=+1]
 
-include::faq.adoc[]
+[appendix]
```

Comment:
How do we deal with this for downstream? Do we need to have some `ifdef` around this?

Reply:
No, the FAQ is [included downstream](https://access.redhat.com/documentation/en-us/red_hat_amq_streams/1.0/html/using_amq_streams_on_openshift_container_platform/frequently_asked_questions). 

I moved the `[appendix]` because I don't think it's for the `faq.adoc` to know that it's an appendix -- it's up the assembling document to decide that.